Frontend/ui-driver/src/Flow.purs

```diff
@@ -130,6 +131,8 @@ import Data.Array as DA
 
 baseAppFlow :: Boolean -> Maybe Event -> FlowBT String Unit
 baseAppFlow baseFlow event = do
+    _ <- lift $lift $ liftFlow $ Events.endMeasuringDuration "Flow.mainFlow"
```

Comment:
@khuzema786 can we replace these -

`_ <- lift $lift $ liftFlow`

with 

`liftFlowBT`